ddp-workspace/projects/ddp-sdk/src/lib/services/cookiesManagement.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Inject, Injectable } from '@angular/core';
+import { CookieService } from 'ngx-cookie';
+import { BehaviorSubject } from 'rxjs';
+import { ConfigurationService } from './configuration.service';
+import { AnalyticsEventsService } from './analyticsEvents.service';
+
+@Injectable()
+export class CookiesManagementService {
+  private isFirstTimeVisitor: BehaviorSubject<boolean> =  new BehaviorSubject(null);
+  private isCookiesToReject: BehaviorSubject<boolean> =  new BehaviorSubject(null);
+  private expireDays = 10;
```

Comment:
I think we decided to put these in local storage instead so there isn't an expiration day.

Reply:
Done in c8a9d8a

Now using local storage instead of cookies